src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -35,6 +35,10 @@ namespace AppInstaller::Logging
         // Used to disable telemetry on the fly.
         std::atomic_bool s_isTelemetryEnabled{ true };
 
+        std::string s_executionStage;
+
+        std::atomic<GUID> s_subExecutionId{ GUID_NULL };
```

Comment:
>std::atomic<GUID> s_subExecutionId{ GUID_NULL }; [](start = 8, length = 48)

Could we instead just use an integer?  These don't need to be unique across sessions, so for the same ActivityId, 0 can be the root (like GUID_NULL), and 1+ can be a subexecution.  That way we also get an order that might be useful.